### ISSUE TITLE
CLI progress bar overflows on queries running for 100+ minutes

### ISSUE THREAD

**Issue by electrum** [prestodb/presto#6293]
Here's an example from a query that's been running for 2+ hours. The CLI terminal width is 80 characters, but this line is rendered at 81 characters (due to the extra minute digit):

```
125:36 [24.9M rows, 3.54GB] [ 3.3K rows/s,  493KB/s] [=====================>] 99%
```

**Comment by electrum** [comment]
Let's cut the progress bar.

Having tests for `StatusPrinter` would be great! That will need a bit of refactoring and should be done separately. We should separate out the printing code from the code that does the update loop and accesses the client.

**Comment by rschlussel** [comment]
fixed in #7380
